File: fizzbuzz.py
# Language Python (http://python.org)
# To Run:
# $ python fizzbuzz.py

# Ultra Simple Fizz Buzz
for n in range(1, 101):
    if n % 3 == 0 and n % 5 == 0:
        print "FizzBuzz"
    elif n % 3 == 0:
        print "Fizz"
    elif n % 5 == 0:
        print "Buzz"
    else:
        print n

print "--- One-Liner ---"
# Condensed One-Liner
print"\n".join(["","Fizz"][f]+["","Buzz"][b]if f+b else str(x)for(x,f,b)in[(x,x%3==0,x%5==0)for x in range(1,101)])

File: fizzbuzz.java
/* To Run:
 * $ javac fizzbuzz.java && java fizzbuzz */

public class fizzbuzz {

    public static void main(String[] argv) {
        for (int i = 0; i <= 100; i++) {
            System.out.print(Integer.toString(i) + " ");
            if (i % 3 == 0) {
                System.out.print("Fizz");
            }
            if (i % 5 == 0) {
                System.out.print("Buzz");
            }
            System.out.print("\n");
        }
    }

}
